t/ui/21-admin-needles.t

```diff
@@ -150,17 +165,17 @@ subtest 'dereference symlink when displaying needles info' => sub {
             t_created              => time2str('%Y-%m-%d %H:%M:%S', time - 200000),
             t_updated              => time2str('%Y-%m-%d %H:%M:%S', time - 200000),
         });
+    push @need_deleted_needles, $real_needle;
+
     my $real_needle_id         = $real_needle->id;
     my $last_seen_module_id    = $real_needle->last_seen_module_id;
     my $last_matched_module_id = $real_needle->last_matched_module_id;
 
     goto_admin_needle_table();
-    $driver->find_element('.dataTables_filter input')->send_keys("bootloader");
-    wait_for_ajax;
     my @needle_trs = $driver->find_elements('#needles tbody tr');
-    is(scalar(@needle_trs), 2, 'two added needles shown');
-    my @symlink_needle_tds = $driver->find_child_elements($needle_trs[1], 'td', 'css');
-    is((shift @symlink_needle_tds)->get_text(), 'symlink_needle_dir', 'symlink needle dir is displayed correctly');
+    is(scalar(@needle_trs), 10, 'two added needles shown');
+    my @symlink_needle_tds = $driver->find_child_elements($needle_trs[9], 'td', 'css');
+    is((shift @symlink_needle_tds)->get_text(), '1symlink_needle_dir', 'symlink needle dir is displayed correctly');
```

Comment:
what does "1symlink_needle_dir" mean?

Reply:
Because the needles are sorted by `Directory` in the table, I set the directory name start with `1` to make sure the first page only show the symlink needles.